controllers/DefaultController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class DefaultController extends BaseEventTypeController
```

Comment:
By reading the JIRA description, it seems like this module is handling external authentication, so does this module really need a DefaultController? If it does need one, the views appear missing. (If those are for later dev work, please ignore me)

Reply:
I removed the DefaultController